index.html

```diff
@@ -0,0 +1,99 @@
+<head>
+    <title>allthingsskin</title>
+</head>
+
+<body>
+
+    <h1>All Things Skin</h1>
+
+    <form>
+        <textarea></textarea>
+    </form>
+
+    <button>Book Now</button>
+    <button>Request a Consultation</button>
+
+    <nav>
+        <ul>
```

Comment:
Is there a reason why you chose to use <ul> elements here with no <li> elements?